lib/rubocop/cop/style/redundant_heredoc_delimiter_quotes.rb

```diff
@@ -22,7 +22,8 @@ module Style
       #   EOS
       #
       #   do_something(<<~'EOS')
-      #     escaped character\.
+      #     Preserve \
```

Comment:
I decided to tweak the example because I think it's more common to use quoted heredocs to retain newlines via `\`.